delocate/cmd/delocate_fuse.py

```diff
@@ -24,18 +23,21 @@
     action="store",
     type=str,
     help="Directory to store delocated wheels"
-    " (default is to overwrite 1st WHEEL input with 2nd)",
+    " (default is to store in the same directory as the 1st WHEEL with an"
+    " automatically determined name).",
 )
 
 
 def main() -> None:  # noqa: D103
     args = parser.parse_args()
     verbosity_config(args)
-    wheel1, wheel2 = [abspath(expanduser(wheel)) for wheel in args.wheels]
-    if args.wheel_dir is None:
-        out_wheel = wheel1
+    wheel1, wheel2 = [Path(wheel).resolve(strict=True) for wheel in args.wheels]
+    if args.wheel_dir is not None:
+        out_wheel = Path(args.wheel_dir).resolve()
+        if not out_wheel.exists():
+            out_wheel.mkdir(parents=True)
```

Comment:
This can be simplified.
```suggestion
        out_wheel.mkdir(parents=True, exist_ok=True)
```